lib/CoreBot.js

```diff
@@ -228,8 +228,31 @@ function Botkit(configuration) {
 
 
         this.ask = function(message, cb, capture_options) {
+          if (message.quick_replies !== undefined) {
```

Comment:
No. It should be undefined. When a message has quick replies, message.quick_replies will not be undefined. When a message does not have quick replies, message.quick_replies will undefined. It won't be a string